### PR TITLE
Changes for single relay P2P release.

### LINKED ISSUE #4193: Clarify where to get peers from for the node
The [understanging-config-files.md](https://github.com/input-output-hk/cardano-node/blob/master/doc/getting-started/understanding-config-files.md) sais:

> Your block-producing node must ONLY talk to your relay nodes, and the relay
> node should talk to other relay nodes in the network. Go to our telegram
> channel to find out IP addresses and ports of peers.

We should clarify how to get peers.

### PR DESCRIPTION
**Pull request by coot**
Requires:

* [x] input-output-hk/ouroboros-network#3859 to be merged
* [x] input-output-hk/ouroboros-network#3891 to be merged
* [x] update `ouroboros-network` hash once the above PRs are merged

- Removed P2PWarningDevelopmentNetworkProtocols
- Modify P2PWarning message
- Updated ouroboros-network dependency

### CODE REVIEW

File: cardano-node/src/Cardano/Node/Startup.hs [hunk 1]
```diff
@@ -97,11 +97,6 @@ data StartupTrace blk =
   -- | Warn when 'EnableP2P' is set.
   | P2PWarning
 
-  -- | Warn that peer-to-peer requires
```
Comment (Jimbo4350) [inline cardano-node/src/Cardano/Node/Startup.hs:100]: Is this deprecated? What's the story here

File: doc/getting-started/understanding-config-files.md [hunk 2]
```diff
@@ -102,6 +103,17 @@ signal to the `cardano-node` process, e.g. `pkill -HUP cardano-node`. After rece
 signal, `cardano-node` will re-read the file and restart all dns resolution. Please
 **note** that this only applies to the topology configuration file!
 
+If you are synchronizing the network for the first time, please connect to IOG
+relays `relays.cardano-node.iohk.io` by adding it to your local root peers and
+set `useLedgerAfterSlot` to `0` (to disable ledger peers).   You can use
```
Comment (Jimbo4350) [inline doc/getting-started/understanding-config-files.md:108]: Where is `useLedgerAfterSlot` being set?

**Review by Jimbo4350 — DISMISSED** [review]

File: cardano-node/src/Cardano/Node/Startup.hs (hunk 1 above)
Comment (coot) [reply cardano-node/src/Cardano/Node/Startup.hs:100]: P2P does not require to experimental protocols.  To set P2P it is enough to set `EnableP2P: True` (one does not need `TestEnableDevelopmentNetworkProtocols: True`)

File: doc/getting-started/understanding-config-files.md (hunk 2 above)
Comment (coot) [reply doc/getting-started/understanding-config-files.md:108]: That section of the documentation describes the p2p topology file format.  `useLedgerAfterSlot` is part of that.

**Review by newhoggy — APPROVED** [review]

**Comment by coot** [comment]
Closing, P2P changes will be postponed.  I will write another PR which updates `cardano-node` which includes this input-output-hk/ouroboros-network#3975. The P2P changes (publishing of `NodeToNodeV_10`) were reverted to avoid having broken `master` branch if `Conway` is released.

**Pull request closed without merging**